tests/integration/security/ca_custom_root/secure_naming_test.go

```diff
@@ -119,8 +119,8 @@ func TestSecureNaming(t *testing.T) {
 			}, retry.Delay(time.Second), retry.Timeout(10*time.Second))
 			var a, b echo.Instance
 			echoboot.NewBuilder(ctx).
-				With(&a, util.EchoConfig("a", testNamespace, false, nil)).
-				With(&b, util.EchoConfig("b", testNamespace, false, nil)).
+				With(&a, util.EchoConfig("a", testNamespace, false, nil, nil)).
```

Comment:
The secure_naming_test and main_test could probably share echos, saving a lot of test time.

Reply:
Oh, converting those aren't part of this PR.